### PR TITLE
Getting started fixes

### PR DESCRIPTION
**Pull request by til**
A few different minor changes that are related to getting started with developing on this project, that 
@dcarral and I found at the eurucamp #pairwithme session.

If you agree with some changes, and not with others, I can also split it in several PRs if you want.

### CODE REVIEW

**Comment by skade** [comment]
I'm not sure about the reason behind `thin` -> `unicorn`

**Comment by til** [comment]
The README says to run foreman, and the Procfile tries to start unicorn. Without having unicorn in the Gemfile, the instructions to get started resulted in an error, so this was an attempt to fix that.

We didn't want to change the Procfile in order to not interfere with existing production setups. Another option would be to make the Procfile environment specific, that uses unicorn only in production. What do you think are the downsides of running unicorn locally for development?

**Comment by skade** [comment]
`!!` that's reasonable! :)

**Comment by til** [comment]
:smile:

File: Gemfile [hunk 1]
```diff
@@ -1,10 +1,8 @@
 source 'https://rubygems.org'
 
-ruby "2.1.5"
```
Comment (myabc) [inline Gemfile]: :+1: for getting rid of this!

**Pull request merged**